templates/demo-store/app/root.tsx

```diff
@@ -83,7 +85,7 @@ export async function loader({request, context}: LoaderFunctionArgs) {
       isLoggedIn: isLoggedInPromise,
       layout,
       selectedLocale: storefront.i18n,
-      cart: cart.get(),
+      cart: cartPromise,
```

Comment:
Should this be reverted now that `customerAccount` is part of cart?

Reply:
I put it above so the code look more like the skeleton code